### ISSUE TITLE
E492: Not an editor command: Neomake

### ISSUE THREAD

**Issue by sergueif** [luan/vimfiles#94]
I ran through the readme using brew-installed regular vim and got this error.

Maybe can be reproduced by doing:
- on macOS 10.12.4
- `brew install vim`
- `curl vimfiles.luan.sh/install | bash`
- try to `:w` a `.yml` file
- observe the above `Neomake` error

Update: I later installed `nvim` and using nvim doesn't suffer from the above. But the Readme claims vanilla vim is also supported.

**Comment by luan** [comment]
Hi @sergueif!

Vanilla vim is not only supported but also it's the one I use most of the time. So it should definitely work. What version of vim is that that you have from brew?

**Comment by sergueif** [comment]
Hi @luan! :)

Ok, I tried again on my older macbook and got other errors:
- `Homebrew 1.2.0`
- `macOS 10.11.6`
```
VIM - Vi IMproved 8.0 (2016 Sep 12, compiled May 15 2017 19:17:19)
MacOS X (unix) version
Included patches: 1-596
Compiled by Homebrew
```

Did
```
brew update && brew upgrade
brew install vim --with-lua
curl vimfiles.luan.sh/install | bash
vim some_new_filename.yml
```

Got:
`Error detected while processing /Users/sergueifilimonov/.vim/plugged/vim-textobj-underscore/plugin/textobj/underscore.vim` (Maybe it failed to clone?)

Then it showed:
`Error detected while processing FileType Auto commands for "yaml":
E492: Not an editor command: :RainbowToggleOff`

After a `:w` I got 
`Error detected while processing BufWritePost Auto commands for "*":
E492: Not an editor command: Neomake`

BUT!! 

I figured I'd run `vim-update` after running the first `curl vimfiles.luan.sh/install | bash` and all those errors went away.

My theory then is that after `curl vimfiles.luan.sh/install | bash`, one has to run `vim-update`. I didn't see that in the Readme, so maybe that's what's needed.
